console/src/state/resources.rs

```diff
@@ -36,7 +36,15 @@ pub(crate) enum SortBy {
 
 #[derive(Debug)]
 pub(crate) struct Resource {
-    id: u64,
+    /// The resource's pretty (console-generated, sequential) ID.
+    ///
+    /// This is NOT the `tracing::span::Id` for the resource's `tracing` span on the
+    /// remote.
+    num: u64,
+    /// The `tracing::span::Id` on the remote process for this resource's span.
+    ///
+    /// This is used when requesting a resource details stream.
+    span_id: u64,
```

Comment:
maybe we can rename it to just pretty_id or display_id?

Reply:
i'm also fine with a name like that; i went with `num` because they're sequential numbers, but I'd be happy to change it.